src/main/java/bio/terra/cli/service/utils/SamService.java

```diff
@@ -67,34 +68,17 @@ private void buildClientForTerraUser() {
     }
   }
 
-  /**
-   * Call the SAM "/version" endpoint to get the version of the server that is currently running.
-   *
-   * @return the SAM version object
-   */
-  public SamVersion getVersion() {
-    VersionApi versionApi = new VersionApi(apiClient);
-    SamVersion samVersion = null;
-    try {
-      samVersion = versionApi.samVersion();
-    } catch (Exception ex) {
-      logger.error("Error getting SAM version", ex);
-    } finally {
-      closeConnectionPool();
-    }
-    return samVersion;
-  }
-
   /**
    * Call the SAM "/status" endpoint to get the status of the server.
    *
-   * @return the SAM status object
+   * @return the SAM status object, null if there was an error checking the status
    */
   public SystemStatus getStatus() {
     StatusApi statusApi = new StatusApi(apiClient);
     SystemStatus status = null;
     try {
-      status = statusApi.getSystemStatus();
+      status =
+          HttpUtils.callWithRetries(() -> statusApi.getSystemStatus(), (ex) -> isRetryable(ex));
```

Comment:
I *think* we can replace the lambda `(ex) -> isRetryable(ex)` with the double colon operator method reference `SamService::isRetryable`

Reply:
Done.